react/components/SpecificationsTranslation/SpecificationsForm.tsx

```diff
@@ -21,7 +21,6 @@ interface SpecificationsFormProps {
 const SpecificationsForm: FC<SpecificationsFormProps> = ({
   specificationInfo,
   specificationId,
-  updateMemoSpecifications,
```

Comment:
We should call this function in order to update the local state. Otherwise, if user navigates to another locale and come back to the one we changed, they will not see the newest value. Also, there is something preventing the success alert to be called. It would be nice to have that solved, since it's part of the same flow.

Reply:
@filafb it is solved now on the latest commit :D